mssql_test.go

```diff
@@ -1917,3 +1918,71 @@ IF @@ROWCOUNT = 0
 		t.Fatal(err)
 	}
 }
+
+func TestMSSQLQueryContextTimeout(t *testing.T) {
+	db, sc, err := mssqlConnect()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer closeDB(t, db, sc, sc)
+
+	testingIssue5 = false
```

Comment:
to be explicit that that is not part of what I was testing.

Reply:
Setting testingIssue5 to false is confusing here, because testingIssue5 is already set to false here. So code reader would be looking for some hidden meaning.

If you want to say something here, just add whatever comment you require.